test/backoff.js

```diff
@@ -47,4 +64,14 @@ describe('backoff', () => {
       expect(res()).to.equal(5)
     )
   })
+
+  describe('when backoff predicate supplied', () => {
+    beforeEach(() =>
+      conditional('a').catch(res)
+    )
+
+    it('retries only when predicate passes', () =>
+      expect(res()).to.equal(3)
```

Comment:
Might be missing something, but i don't see where the number of retries is tested?

Reply:
That's the `when: gte(2)` part.  Only retry when `2` is greater-or-equal to the "error".  Even though `tries` is set to 5, the `when` should limit it to 3, which is what that tests.